texmf/tex/latex/fks/fkssugar.sty

```diff
@@ -87,6 +87,14 @@
 \newcommand\op[1]{\mathup{#1}}                              % operators
 \newcommand\abs[1]{\left|#1\right|}                         % absolutní hodnota
 \newcommand\conj[1]{#1^\ast}                                % komplexní sdružení
+\newcommand\trans{\mathsf{T}}                               % transpozice matice (není použito ^\mathsf{T} proto, že to by potom nešlo složit s jinými horními indexy)
```

Comment:
Když je to stejně velké, jak ta matice, tak to nevypadá dobře. Nešlo by problém s více horními indexy vyřešit skupinou? Něco jako 
```
\newcommand\trans[1]{{#1}^{\mathsf{T}}}
```

Reply:
Problém by byl v tom, že pak by to vypadalo jako na horním obrázku, zatímco chceme, aby to bylo jako na tom dolním.
![image](https://user-images.githubusercontent.com/43704297/105578679-328a7b00-5d82-11eb-9da6-17492d50001c.png)

Ale napadlo mě ještě jedno řešení - co zavést pro každý takový symbol (ještě mě napadá \conj, což vyjadřuje komplexní sdružení, teda hvězdičku) dvě podoby. Jedna bude s velkým prvním písmenem (např. \Trans) a bude vyjadřovat jen ten symbol. Druhá pak bude s malým (\trans) a bude to buď ^{\Trans} nebo #1^{\Trans}. Pointou je, že většinou chceš použít jenom jednu věc a potom double superscipt nehrozí. Na druhou stranu, když chceš, máš tu možnost použít i přímo jen ten symbol. Co si o tom myslíte?

Reply:
Nevím, zda to je potřeba, podle mě potřebujeme jen `^{\trans}` -- tj. symbol používáme jen jako horní index, separátně mě nenapadá, co s tím. Tedy mi vyhovuje současný stav.

Reply:
Ale to s tou stříškou funguje dobře, ne?

    \newcommand\trans{^\mathsf{T}}  
    A\trans\trans

ti udělá:

![image](https://user-images.githubusercontent.com/23003372/105579721-7b453280-5d88-11eb-8259-9dfdf788e0fa.png)

Reply:
Mě to nefunguje
![image](https://user-images.githubusercontent.com/43704297/105579820-29e97300-5d89-11eb-9b41-3d7f955753bf.png)

Reply:
Jasně, díky za info. Zkoušel jsem to v Overleafu a místo chyby to nějak maskoval. Tak co toto?

`\newcommand\trans{{}^\mathsf{T}} `

Reply:
to nejspíš nebude fungovat, když budeme chtít aplikovat transpozici na nějakou větší závorku (což jsou třeba matice, které tu teď simuluju zlomkem)
![image](https://user-images.githubusercontent.com/43704297/105580137-e7c13100-5d8a-11eb-9024-115391232536.png)